# We get the pose as (x,y,theta)
float64 x
float64 y
float64 theta
---
# respond with the new pose for confirmation
geometry_msgs/Pose new_pose
# can be either success or an error shown to the client
string outcome 
